### DISCUSSION TITLE
Place to put SkotOS/library feature requests

### DISCUSSION THREAD

**Discussion by noahgibbs** [ChatTheatre/Community#15] [Ideas]
Hey, folks.

Some feature requests have an obvious repository to put them in. For instance if you wanted Orchil to support a different colour scheme setup, you'd pretty clearly put that feature request into the Orchil repo.

Some feature requests kind of have a location but not really: if you wanted the shared SkotOS clothing Ur-objects to support a "kibitz" action where you complained about the quality of tailoring in great detail, presumably that feature request would wind up in the SkotOS repo with those shared Ur-objects. We're preparing to take more game-type stuff *out* of SkotOS, but for now it would go there.

But many feature requests don't map cleanly to an existing repo. Want to suggest creating a new Grand Theatre where the characters are cows, goats and other dairy animals and all combat is cheese-based? That's not an obvious match for the SkotOS repo, but it certainly isn't a better match for anything currently under github.com/ChatTheatre.

Similarly, some underlying features are more general than SkotOS, which means we don't have an obvious place for them. I was looking at filing an issue for a JSON parser/emitter library, roughly similar to the current SkotOS XML parser/emitter library. But there's no reason for that to be SkotOS-specific. In fact it would be much better if it weren't.

If nobody objects, I'm going to start filing those issues here in ChatTheatre/Community.

**Comment by ChristopherA** [comment]
Sounds good to me.
